Code/GraphMol/SubstructLibrary/PatternFactory.cpp

```diff
@@ -19,22 +19,37 @@
 namespace RDKit {
 namespace {
 void fillPatterns(const SubstructLibrary &slib,
-		  const PatternHolder &fph,
+                  const FPHolderBase &fph,
 		  std::vector<ExplicitBitVect*> &fps,
 		  unsigned int start, unsigned int end, unsigned int numThreads) {
   for (unsigned int idx = start;
        idx < end;
        idx += numThreads) {
-    fps[idx] = fph.makeFingerprint( *slib.getMol(idx).get() );
+    auto mol = slib.getMol(idx);
+    if (mol.get()) {
+        fps[idx] = fph.makeFingerprint( *mol.get() );
+    }
+    else {
```

Comment:
```suggestion
    } else {
```
(general aside: wouldn't be bad to run clang-format pre-submission)

Reply:
Now that #3798 is accepted I’ll merge with master.

I’ll add the extra tests, I misunderstood your previous comment.